### DISCUSSION TITLE
An easy way to disable Save feature in all editors?

### DISCUSSION THREAD

**Discussion by murty2** [DustinBrett/daedalOS#443] [Q&A]
Is there an easy way to disable Save feature in all editors?

Right now, for Monaco, I changed in components/apps/MonacoEditor/StatusBar.tsx
`await writeFile(saveUrl, saveData, true);`
to
`await writeFile('', saveData, true);`

Is there a way to set a global variable or change writeFS to disable save feature in all editors: Monaco, TinyMCE, Marked etc.. ?

**Comment by DustinBrett** [comment]
I don't think there is a global way. It was custom added for each app to be able to save. You'd need to manually remove the code and buttons.
